Assets/Mirror/Components/NetworkRoomManager.cs

```diff
@@ -317,21 +324,36 @@ public override void OnServerAddPlayer(NetworkConnection conn)
 
             if (IsSceneActive(RoomScene))
             {
-                if (roomSlots.Count == maxConnections)
-                    return;
+                CreateRoomPlayer(conn);
+            }
+            else if (AllowJoinGameInProgress && conn.identity == null)
+            {
+                CreateRoomPlayer(conn);
+                SceneLoadedForPlayer(conn, conn.identity.gameObject);
+                OnClientSceneChanged(conn);
```

Comment:
OnClientSceneChanged isn't necessary, but I thought it would be best to call it so it would call any hooks developers have made.